tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -62,14 +62,11 @@ def get_framework_results(self, inputs_dict, model_path):
                 tf.compat.v1.import_graph_def(graph_def, name='')
 
                 input = dict()
-                if self.api_2:
-                    input.update(inputs_dict)
-                else:
-                    for key in inputs_dict.keys():
-                        data = inputs_dict.get(key)
-                        if not self.api_2:
-                            key += ':0'
-                        input[key] = transpose_nchw_to_nhwc(data, self.use_new_frontend, self.api_2)
+                for key in inputs_dict.keys():
+                    data = inputs_dict.get(key)
+                    if not self.api_2 or self.use_new_frontend:
+                        key += ':0'
```

Comment:
@rkazants do you agree with this?

Reply:
@jane-intel, agree - looks a bit strange. @rnugmanx, please elaborate why is this suffix needed

Reply:
@rkazants with new TF FE IR has two names for input layer ('Input' and 'Input:0' for example) while usual scenario generates IR with only one name for input layer ('Input:0') - is it ok?

Reply:
I see, it relates to one output operation. What if it has two outputs, will the tensor name of the second output contain this suffix.
I think it is ok that TF FE has two tensor names for the first output, we can you any which is convinient.